kauai/SRC/UTIL.H

```diff
@@ -91,7 +92,7 @@ const long klwMin = -klwMax;	// so -klwMin is positive
 const ulong kluMax = 0xFFFFFFFF;
 const ulong kluMin = 0;
 
-typedef int bool;
+#define bool int
```

Comment:
I feel like this should be marked so it could be cleaned up later, in ScummVM we would do a comment like

```
// HACK: redefining bool
```

or FIXME would also work, we should probably decide on one as a convention so they're easy to search for